contracts/RandomAuRa.sol

```diff
@@ -66,6 +66,7 @@ contract RandomAuRa is RandomBase, IRandomAuRa {
     function initialize(
         uint256 _collectRoundLength // in blocks
     ) external {
+        require(block.number == 0);
```

Comment:
Let's also add here `require(_collectRoundLength > 0);` and `require(collectRoundLength() == 0)`.

Reply:
Agreed.